woof-code/support/busybox_symlinks.sh

```diff
@@ -100,7 +100,7 @@ do
 	esac
 	#-
 
-	if [ -f bin/$N -o -f sbin/$N -o -f usr/bin/$N -o -f usr/sbin/$N ] ;then
+	if [ -e bin/$N -o -e sbin/$N -o -e usr/bin/$N -o -e usr/sbin/$N ] ;then
```

Comment:
This change prevents collision with other `vi`s like vim, where /usr/bin/vi may be a symlink

Reply:
Makes sense